impress-org/givewp
# Fix: Handle empty excerpt field for v3 forms on the Form Grid

## Description

While rendering a v3 form on the Form Grid. The excerpt field displays an empty array.

This pull request replaces the array which is represented as a string '[]'  with an empty string so that it no longer displays in the form grid.

## Affects

Form Grid

## Visuals

<img width="1725" alt="Screen Shot 2023-06-26 at 10 41 30 AM" src="https://github.com/impress-org/givewp/assets/75056371/45024e76-5de9-4acd-879d-f15a8597aa59">

## Testing Instructions

- Create a v3 donation form
- Create a page using the form grid
- Verify that '[]' does not display on the form grid

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204982899163702
